### PR TITLE
Hotfix/snap zone highlighting

### PR DESCRIPTION
**Pull request by Gusinuhe**
### Description

**Fixes**: 

- Snap Zones now render highlight objects with sub meshes.
- Snap Zone Settings creates transparent materials.

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by Maxsiom** [comment]
image

Works perfectly for me :)

**Review by Maxsiom — APPROVED** [review]

File: Runtime/Interaction/SnapZone.cs [hunk 1]
```diff
@@ -131,19 +130,20 @@ public Material InvalidMaterial
         /// </summary>
         protected XRBaseInteractable ForceSelectTarget { get; set; }
         
-        private Mesh previewMesh;
+        private Mesh previewMeshes;
         
-        internal Mesh PreviewMesh 
+        internal Mesh PreviewMeshes
```
Comment (tomwim) [inline Runtime/Interaction/SnapZone.cs:135]: Not sure if I like this internal tbh. Not being able to get access to members is one of the most annoying things in Unity XR

**Review by tomwim — CHANGES_REQUESTED** [review]

File: Runtime/Interaction/SnapZone.cs (hunk 1 above)
Comment (Gusinuhe) [reply Runtime/Interaction/SnapZone.cs:135]: Why would you want to access this member?

**Comment by SimonTheSourcerer** [comment]
reopend this PR in cleaned up state: 
https://github.com/Innoactive/XR-Interaction-Component/pull/51

**Pull request closed without merging**